flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -75,6 +76,9 @@
 	private static final String METRIC_OUTPUT_POOL_USAGE = "outPoolUsage";
 	private static final String METRIC_INPUT_QUEUE_LENGTH = "inputQueueLength";
 	private static final String METRIC_INPUT_POOL_USAGE = "inPoolUsage";
+	private static final String METRIC_CREDIT_BASED_FLOATING_BUFFER_POOL_USAGE = "floatingBufferUsage";
```

Comment:
I think it is better to not add the keywords `CREDIT-BASED` here, because after we remove the deprecated network codes finally, it is no need to emphasize this term any more. The terms of exclusive/floating already indicate that.
Also it should be consistent between variable name and value, like `METRIC_FLOATING_BUFFERS_USAGE = "floatingBuffersUsage"`